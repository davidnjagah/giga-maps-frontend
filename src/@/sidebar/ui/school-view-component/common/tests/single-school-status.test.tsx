import { render } from '@testing-library/react';
import { testWrapper } from '~/tests/jest-wrapper';
import SingleSchoolStatus from '../single-school-status-view';

describe('Single school status', () => {
  it('should render component', () => {
    const { asFragment } = render(testWrapper(<SingleSchoolStatus />));
    expect(asFragment).toMatchSnapshot();
  });
});
