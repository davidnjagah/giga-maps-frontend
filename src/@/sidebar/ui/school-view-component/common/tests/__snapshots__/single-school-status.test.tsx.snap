// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Single school status should render component 1`] = `[Function]`;
